ps_facetedsearch.php

```diff
@@ -411,9 +411,17 @@ public function indexProductPrices($idProduct, $smart = true)
             );
 
             if (empty($taxRatesByCountry) || !Configuration::get('PS_LAYERED_FILTER_PRICE_USETAX')) {
-                $idCountry = (int) Configuration::get('PS_COUNTRY_DEFAULT', null, null, $idShop);
-                $isoCode = Country::getIsoById($idCountry);
-                $taxRatesByCountry = [['rate' => 0, 'id_country' => $idCountry, 'iso_code' => $isoCode]];
+                $shopCountries = Country::getCountriesByIdShop($idShop, $this->getContext()->language->id);
+                $taxCountries = array_filter($shopCountries, function ($country) {
+                    return $country['active'] == 1;
```

Comment:
```suggestion
                    return $country['active'];
```
The comparison with `==` are useless